### ISSUE TITLE
Make print media (e.g. flyer, poster) easily available

### ISSUE THREAD

**Issue by DougInAMug** [inventaire/inventaire#348]
While traveling recently I distributed some stickers to areas where people haven't yet hear The Good News™ about Inventaire.

However there were and are many places which would happily give prime wall space to propagandise. Sadly I couldn't find an Inventaire flyer poster.

Does one exist already? If not, is there material which could be easily hacked into one?

**Comment by maxlath** [comment]
we have some on the roadmap trello thingy
* [presentation flyers](https://trello.com/c/4mXofMDy/140-flyers-posters)
* [local library QR code flyer](https://trello.com/c/779FYGzp/144-local-library-flyer)
* [stickers](https://trello.com/c/FT74DA2U/28-stickers)

**Comment by black-puppydog** [comment]
Just stumbled over this, and I'm not sure if this is addressed by the flyers/stickers here?
https://inventaire.github.io/communication/
Sorry that isn't it, it's a bit hard to tell whether this issue predates that site 🙂

**Comment by maxlath** [comment]
@black-puppydog right! I think the website predates this issue, I don't know why i forgot about it (maybe because we don't use it enough ><). It makes me think, a few things that could be done
- [  ] add A4 collage to make it easier to print many flyers
- [  ] recover a grey scale version of the flyer to allow people to go easy on their ink budget (we had one, but it got obsolete at some point)
- [  ]  make an hexagonal sticker, as NLnet offered to print a bunch
